### DISCUSSION TITLE
raylib game deploy to other distributions

### DISCUSSION THREAD

**Discussion by 2gud4u** [raysan5/raylib#2795] [Q&A] [answered]
Hello community.-

I have asked this question in the past but so far no one could anwer it. I have a game named engine and i want to distribute to other Linux distributions, when they execute the game this error is shown `./engine: /lib/x86_64-linux-gnu/[libm.so](http://libm.so/).6: version GLIBC_2.35' not found (required by ./engine) ./engine: /lib/x86_64-linux-gnu/libc.so.6: version GLIBC_2.33' not found (required by ./engine)
./engine: /lib/x86_64-linux-gnu/[libc.so](http://libc.so/).6: version GLIBC_2.34' not found (required by ./engine)`. I have also tried to pack my application/game with AppImage but when they install in the target OS this error is shown again. Could someone tell me how can i distribute my game to other Linux distros ? I compile link with this command `gcc -o engine main.c -lraylib -lGL -lm -lpthread -ldl -lrt -lX11 -lstdc++ -lc` As suggested i have added the -lc flag, which it should include libc.so. Thanks in advance !!

**Comment by gen2brain** [comment, marked as answer]
You can try to compile with `-static-libgcc` and  `-static-libstdc++`. The simplest solution would be to just use some old LTS Ubuntu or similar with some older Glibc and compile it there.

Musl toolchain or fully `--static` Glibc build would be hard or impossible because of OpenGL, maybe only with software rendering.

**Reply by 2gud4u** [reply]
So i tried to compile with -static-libgcc and also -static-libstdc++ and i got the same error on my Debian virtual machine, its a shame that i have to go to another Linux distro to get to run my game "exe". @gen2brain do you know if with another Arch linux distribution would be the same error ? I think yes maybe because its also a rolling distro right ? Which distro do you recommend me for compiling ? Thank you
